content/testing-yaml/testing.md

```diff
@@ -31,13 +31,13 @@ scripts:
 
 ## Flutter integration tests
 
-The `flutter_driver` dependency allows you to run integration tests on a real device or emulator. Android application tests can be run on an Android emulator, iOS application tests can be run on an iOS simulator, and web application tests can be run on a web browser driver.
+The `integration_test` dependency allows you to run integration tests on a real device or emulator. Android application tests can be run on an Android emulator, iOS application tests can be run on an iOS simulator, and web application tests can be run on a web browser driver.
 
 ### Running iOS/Android application tests on a mobile simulator/emulator
 
 ```bash
 flutter emulators --launch apple_ios_simulator             # for android use: flutter emulators --launch emulator
-flutter drive --target=test_driver/my_drive_target.dart
+flutter drive --driver=test_driver/integration_driver.dart --target=integration_test/app_test.dart -d <device_id>
```

Comment:
I'm not sure what the device ID should be on our system

Reply:
looks like what we have available on machines are these devices: https://docs.codemagic.io/testing/running-automated-tests/#running-flutter-driver-tests

Reply:
Updated the PR to mention device_id for both platforms

In CM machine from ssh, shows `emulator-5554` (seme as ID that CM editor adds) and `iPhone` launches simulator 
```
builder@builders-Mac-130 ~ % flutter devices
2 connected devices:

Android SDK built for x86 (mobile) • emulator-5554 • android-x86    • Android 10
(API 29) (emulator)
Chrome (web)                       • chrome        • web-javascript • Google
Chrome 90.0.4430.93
```